apis/v1alpha1/httproute_types.go

```diff
@@ -167,6 +167,20 @@ type HTTPRouteRule struct {
 	// path match on "/", which has the effect of matching every
 	// HTTP request.
 	//
+	//
+	// Although a client request may technically match multiple HTTP route rules,
+	// only one rule may ultimately receive the request. Matching precedence
+	// MUST be determined in order of the following criteria:
+	//
+	// * The most specific match is determined by the longest matching combination
+	//   of hostname and path. In the event of a tie, the larger number of header
+	//   matches will be used to differntiate.
+	// * The oldest Route based on creation timestamp. For example, a Route with
```

Comment:
I'm not sure exactly how to structure this, but both the point above and below start with something saying something like "if still equivalent". Maybe all bullet points should start with that? I could also be fine with none as long as it was consistent.

Reply:
i think ive moved into a better language form in the latest commit marking resolve please nit if this is in error :) !